libpod/runtime_pod_linux.go

```diff
@@ -80,24 +86,23 @@ func (r *Runtime) NewPod(options ...PodCreateOption) (*Pod, error) {
 		return nil, errors.Wrapf(err, "error adding pod to state")
 	}
 
+	if usePause && podIsSharing(pod) {
+		if err := r.CreatePauseContainer(ctx, pod); err != nil {
```

Comment:
Function should probably be private, I don't think we want it called outside of libpod